pyproject.toml

```diff
@@ -49,32 +49,28 @@ build-backend = "setuptools.build_meta"
 packages = ["weave"]
 
 [tool.setuptools.dynamic]
-version = {attr = "weave.version.VERSION"}
-dependencies = {file = ["requirements.txt"]}
+version = { attr = "weave.version.VERSION" }
+dependencies = { file = ["requirements.txt"] }
 
 [tool.setuptools.dynamic.optional-dependencies]
-examples = {file = ["requirements.ecosystem.txt"]}
-engine = {file = ["requirements.engine.txt"]}
-ecosystem = {file = ["requirements.ecosystem.txt"]}
-datadog = {file = ["requirements.datadog.txt"]}
-modal = {file = ["requirements.modal.txt"]}
+examples = { file = ["requirements.ecosystem.txt"] }
+engine = { file = ["requirements.engine.txt"] }
+ecosystem = { file = ["requirements.ecosystem.txt"] }
+datadog = { file = ["requirements.datadog.txt"] }
+modal = { file = ["requirements.modal.txt"] }
 
 [tool.pytest.ini_options]
 testpaths = ["weave"]
 filterwarnings = ["error"]
 
-[tool.autopep8]
-max_line_length = 180
-ignore = "E714"
-
-[tool.ruff]
-line-length = 180
-show-fixes = true
+[tool.ruff.lint]
 select = [
   "FIX001", # https://beta.ruff.rs/docs/rules/#flake8-fixme-fix
   "FIX003",
   "F541",
 ]
 
-[tool.flake8]
-max-line-length = 180
+[tool.ruff]
+line-length = 180
+show-fixes = true
+exclude = ["weave/api.py", "weave/__init__.py", "examples"]
```

Comment:
Just use ruff and don't format `api.py` or `__init__.py` which can cause a lot of problems because of import ordering